src/components/templates/PosterPage/PosterPage.tsx

```diff
@@ -25,9 +26,14 @@ export interface PosterPageProps {
 }
 
 export const PosterPage: React.FC<PosterPageProps> = ({ venue }) => {
-  const { id: venueId, isLive: isPosterLive, poster, iframeUrl } = venue;
+  const {
+    id: venueId,
+    isLive: isPosterLive = false,
```

Comment:
This default shouldn't have been moved out here like this. I've described this in more detail on the previous comment thread about it, but basically since you removed the 'inner component' of the modal, it made my suggested comments irrelevant, and the code was correct before the changes made that implemented this.

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1299/commits/ce562c6a4616118ea904bb8f088ca751787a5f95